samples/matter/common/src/persistent_storage/Kconfig

```diff
@@ -38,8 +38,16 @@ config NCS_SAMPLE_MATTER_SECURE_STORAGE_PSA_KEY_VALUE_OFFSET
 	hex "The PSA key offset dedicated for Matter application"
 	default 0x40000
 
+config NCS_SAMPLE_MATTER_SECURE_STORAGE_BACKEND_MIGRATE_UID_MAP
+	bool "Migrate UID map to Zephyr Settings"
+	default y
+	help
+	  Migrate UID map from secure storage to Zephyr Settings. Storing UID map in
+	  Settings allows to reduce TRUSTED_STORAGE_BACKEND_AEAD_MAX_DATA_SIZE and
+	  stack size requirements for threads accessing secure storage.
+
 config TRUSTED_STORAGE_BACKEND_AEAD_MAX_DATA_SIZE
-	default 1410
+	default 1410 if NCS_SAMPLE_MATTER_SECURE_STORAGE_BACKEND_MIGRATE_UID_MAP
```

Comment:
So we need it only for migration purpose, then after migration, someone should disable this config? It is ok, but we should add migration guide entry for sure, to recommend such a scenario.